enterprise/frontend/src/embedding-sdk/index.ts

```diff
@@ -62,3 +62,5 @@ export type {
   MetabaseFetchRequestTokenFn,
   MetabaseEmbeddingSessionToken,
 } from "./types/refresh-token";
+
+export type { EmbeddingEntityType } from "metabase/embedding-sdk/store.ts";
```

Comment:
[nitpick]

```suggestion
export type { EmbeddingEntityType } from "metabase/embedding-sdk/store";
```

Reply:
awee fkk. I was too occupied with the CI, I missed this.